### PR TITLE
Filter out invalid DHCP servers when detecting crowbar admin server

### PR DESCRIPTION
**Pull request by vuntz**
Octavian Ciuhandu proposed this code.

### CODE REVIEW

**Comment by vuntz** [comment]
Note that I haven't tested this.

File: build_winpe.ps1 [hunk 1]
```diff
@@ -186,7 +186,7 @@ popd
 Add-Content $startnet_cmd "`n"
 if($built_for_crowbar)
 {
-  Add-Content $startnet_cmd "`n powershell `"`$DHCPServer = @(Get-WMIObject -Class Win32_NetworkAdapterConfiguration -Filter IPEnabled=True -ComputerName . | Select-Object -Property DHCPServer).DHCPServer[0] ; Write-Host DHCPServer=`$DHCPServer ; net use $crowbar_mountpoint \\`$DHCPServer\$crowbar_share`""
+  Add-Content $startnet_cmd "`n powershell `"`$DHCPServers=@(gwmi Win32_NetworkAdapterConfiguration -Filter IPEnabled=True | where {`$.DHCPServer -and !(`$.DHCPServer -match `"^255`")}); if (`$DHCPServers) {`$DHCPServer=`$DHCPServers[0].DHCPServer} else {throw `"DHCP Server Not Found`"} ; Write-Host DHCPServer=`$DHCPServer ; net use $crowbar_mountpoint \\`$DHCPServer\$crowbar_share`""
```
Comment (ociuhandu) [inline build_winpe.ps1]: $.DHCPServer should be, in fact, $_.DHCPServer in both cases.

Comment (vuntz) [inline build_winpe.ps1]: Ah, thanks, the `_` got lost in the github formatting on the other PR, that's why I hadn't seen it :-)

**Comment by bmwiedemann** [comment]
successfully tested it here on a node with two interfaces

**Pull request merged**